boot/boot_test.go

```diff
@@ -61,15 +61,15 @@ func (s *baseBootSetSuite) SetUpTest(c *C) {
 type bootSetSuite struct {
 	baseBootSetSuite
 
-	loader *boottest.MockBootloader
+	loader *bootloadertest.MockBootloader
```

Comment:
preexisting to this and can be a follow up, but I'm not super fond of calling a bootloader just "loader" (I know historically that was a practiced terminology but there are too many things to load these days),  bootloader, bloader or bl  or just "b" (it's okish but not my preference), would work better